indexclient/client.py

```diff
@@ -173,10 +175,10 @@ def to_json(self, include_rev=True):
         json["did"] = self.did
         return json
 
-    def refresh(self):
+    def refresh(self, json=None):
```

Comment:
the added parameter breaks the purpose of this function, just don't call refresh for initialization?

Reply:
How does it break it?? The json value isn't cached. I don't want to remove refresh from init some code may be relying on that behavior?

Reply:
aren't you just not calling server to get latest json if a json is passed? basically you don't refresh if a json is passed?

Reply:
If you pass a json it loads it from that json. Its why we still need this function to translate from json -> internal structure. The function could be renamed from refresh to load, but that might break people so its not worth it.

Reply:
nothing else is using it as it serves more like a private method, let's rename it and update docstring to minimize confusion for future?